schema/kbase/jsonschema/credit_metadata.schema.json

```diff
@@ -311,8 +434,22 @@
             "title": "PermanentID",
             "type": "object"
         },
+        "ProviderType": {
+            "description": "The provider of a resource.",
+            "enum": [
+                "Crossref",
+                "DataCite",
+                "GitHub",
+                "JaLC",
+                "KISTI",
+                "mEDRA",
+                "OP"
+            ],
+            "title": "ProviderType",
+            "type": "string"
+        },
         "RelationshipType": {
-            "description": "the relationship between two entities",
+            "description": "The relationship between two entities",
             "enum": [
```

Comment:
I had a deduped list of relationship types, equivalent to the Crosscite relations that you mentioned in your email. I have a feeling that I used that list when creating object models for the schema in another repo, but neglected to update this list.

Reply:
The devil is in the detail :).